package.json

```diff
@@ -139,6 +139,7 @@
     "@kbn/es-query": "link:bazel-bin/packages/kbn-es-query",
     "@kbn/field-types": "link:bazel-bin/packages/kbn-field-types",
     "@kbn/flot-charts": "link:bazel-bin/packages/kbn-flot-charts",
+    "@kbn/handlebars": "link:bazel-bin/packages/kbn-handlebars",
```

Comment:
you have missed here the types pkg on devDeps:

```
"@types/kbn__handlebars": "link:bazel-bin/packages/kbn-handlebars/npm_module_types",
```

Reply:
Nice catch! I didn't know I had to do that as well 👍